ddr/ddr/Models/Effects/HideEffect.swift

```diff
@@ -0,0 +1,16 @@
+
+import Foundation
+import SpriteKit
+
+class HideEffect: Effect {
+  
+  let nodeToHide: SKNode
+  
+  init(nodeToHide: SKNode) {
+    self.nodeToHide = nodeToHide
+  }
+  
+  func apply(to game: GameScene) {
+    nodeToHide.isHidden = true
```

Comment:
Yes, effects may need to alter the game scene, but not all of them do. The protocol is written to allow them to because, in my mind, it is not necessary to distinguish between those that do and do not need the game scene.